### PR TITLE
Rspec feature login

### PR DESCRIPTION
**Pull request by MenBui**
==> Title: rspec_feature_login

==> Redmine: 

==> Attachment File:
Screenshot from 2020-01-02 15-09-20
Screenshot from 2020-01-02 15-02-19

### CODE REVIEW

**Comment by MenBui** [comment]
ready

File: config/settings.yml [hunk 1]
```diff
@@ -38,3 +39,6 @@ rest_amount: 15
 id_user1: 1
 id_user2: 22
 role_user: 2
+=======
```
Comment (manhbnt-0960) [inline config/settings.yml]: .

File: spec/factories/author.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 FactoryBot.define do
   factory :author do
     name {Faker::Name.name}
+<<<<<<< HEAD
     description {Faker::Lorem.sentence(word_count: Settings.word_amount)}
+=======
+    description {Faker::Lorem.sentence(word_count: Settings.word_count)}
+>>>>>>> rspec_model_author
```
Comment (manhbnt-0960) [inline spec/factories/author.rb]: .

**Comment by MenBui** [comment]
ready

File: spec/features/login_spec.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  end
+
+  feature "Login success" do
+    scenario "role: user" do
+      fill_in "session[email]", with: user.email
+      fill_in "session[password]", with: user.password
+      within ".form-login" do
+        click_button "commit"
+      end
+      expect(page).to have_current_path("/en")
```
Comment (manhbnt-0960) [inline spec/features/login_spec.rb]: login fail cũng có /en mà e

**Comment by MenBui** [comment]
ready

**Pull request merged**